CMP1903_A1_2324/Testing.cs

```diff
@@ -16,5 +17,28 @@ internal class Testing
          */
 
         //Method
+        private void testDie()
```

Comment:
Style guidelines state that this should be TestDie(), CamelCase(), refer to naming of methods.